build/mac/build_delta_installer.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import optparse
+import subprocess
+import sys
+
+
+def Main(argv):
+  parser = optparse.OptionParser('%prog [options]')
+  parser.add_option('--binary-delta', dest='binary_delta_path', action='store',
+      type='string', default=None, help='The path of BinaryDelta binary.')
+  parser.add_option('--sign-update', dest='sign_update_path', action='store',
```

Comment:
should add option to pass signing key here instead of expecting it to be in the keychain

Reply:
Done.
`.npmrc` should have below two key.
```
sparkle_eddsa_public_key=xxxxxxxxxxxxxxxxxxx
sparkle_eddsa_private_key=xxxxxxxxxxxxxxxxxxxxx
```